connectivity/manifests/egress-gateway-policy-excluded-cidrs.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: cilium.io/v2
+kind: CiliumEgressGatewayPolicy
+metadata:
+  name: cegp-sample-excluded-cidrs
+spec:
+  selectors:
+  - podSelector:
+      matchLabels:
+        io.kubernetes.pod.namespace: cilium-test
```

Comment:
Do we dynamically change it somewhere later on? Asking, as AFAIK `cilium-test` is configurable.

Reply:
yup, it's already been taken care by WithCiliumEgressGatewayPolicy:

https://github.com/cilium/cilium-cli/blob/44ae1874fae4544c0db34dac89c11e37365b76ef/connectivity/check/test.go#L435-L446